tests/tests/test-parachain-bond.ts

```diff
@@ -0,0 +1,169 @@
+import { expect } from "chai";
+import { Keyring } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { Event } from "@polkadot/types/interfaces";
+import {
+  GENESIS_ACCOUNT,
+  ALITH_PRIV_KEY,
+  GENESIS_ACCOUNT_PRIVATE_KEY,
+  ZERO_ADDRESS,
+} from "../util/constants";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+const TWENTY_PERCENT = 20;
+const TWENTY_PERCENT_STRING = "20.00%";
+
+describeDevMoonbeam("Staking - Parachain Bond - genesis and setParachainBondAccount", (context) => {
+  let sudoAccount: KeyringPair;
+  before("Setup genesis account for substrate", async () => {
+    const keyring = new Keyring({ type: "ethereum" });
+    // genesisAccount = await keyring.addFromUri(GENESIS_ACCOUNT_PRIVATE_KEY, null, "ethereum");
+    sudoAccount = await keyring.addFromUri(ALITH_PRIV_KEY, null, "ethereum");
+  });
+  it("should have right parachain bond in genesis", async function () {
+    const parachainBondInfo = await context.polkadotApi.query.parachainStaking.parachainBondInfo();
+    expect(parachainBondInfo.toHuman()["account"]).to.equal(ZERO_ADDRESS);
+    expect(parachainBondInfo.toHuman()["percent"]).to.equal("30.00%");
```

Comment:
the extrinsic `setParachainBondPercent` sets this percent field (only can be set by sudo)

so you could write another test for this runtime method, but it isn't necessary at all

Reply:
The tests test`setParachainBondReservePercent` isnt thqt the right function to test or do I need to change it to `setParachainBondPercent` ?

Reply:
Yes that is the correct function, I didn't see that test.